### ISSUE TITLE
is client-configuration.yml is only used for the python client?

### ISSUE THREAD

**Issue by julie-sullivan** [opencb/cellbase#452]
We should use the same config files for all clients.

**Comment by imedina** [comment]
Yes, I think so, we need to make _cellbase.sh_ CLI to use client-configuration.yaml. BTW, we should probably review this configuration file.
